invenio_github/utils.py

```diff
@@ -90,8 +90,11 @@ def get_citation_metadata(gh, owner, repo_name, ref, release):
             release.errors = {"CITATION.cff": errors}
         return data
     except ValueError:
-        raise CustomGitHubMetadataError(
```

Comment:
I think we should still be raising an exception here (i.e. if one chooses to get metadata from `CITATION.cff` and it fails they should be responsible for handling any exceptions, or let them bubble up)